deno.json

```diff
@@ -41,6 +43,9 @@
     "node:unit": "DENO_FUTURE_CHECK=1 deno test --unstable --allow-all node/ --ignore=node/_tools/test.ts,node/_tools/versions/",
     "node:test": "deno test --unstable --allow-all node/_tools/test.ts",
     "node:setup": "deno run --allow-read --allow-net --allow-write node/_tools/setup.ts",
-    "node:check-unstable-api": "deno check --unstable ./node/module_all.ts"
+    "node:check-unstable-api": "deno check --unstable ./node/module_all.ts",
+    "build:crypto": "cd _wasm_crypto && deno run --unstable -A --no-check https://deno.land/x/wasmbuild@0.5.0/main.ts --sync && mv lib/deno_std_wasm_crypto.generated.js lib/deno_std_wasm_crypto.generated.mjs",
+    "build:hash": "cd hash/_wasm && deno run --unstable -A --no-check https://deno.land/x/wasmbuild@0.5.0/main.ts --sync && mv lib/deno_hash.generated.js lib/deno_hash.generated.mjs",
+    "build:varint": "cd _wasm_varint && deno run --unstable -A --no-check https://deno.land/x/wasmbuild@0.5.0/main.ts --sync && mv lib/deno_std_wasm_varint.generated.js lib/deno_std_wasm_varint.generated.mjs"
```

Comment:
What would be better to create a single source of truth is something like a `--cwd <dir>` option to say what directory deno task should use when it starts up (I like how it uses the same directory as the deno.json, but then it means in cases like this we can't create an alias).

```json
"build:crypto": "cd _wasm_crypto && deno task --cwd $PWD wasmbuild --sync",
"build:hash": "cd hash/_wasm && deno task --cwd $PWD wasmbuild --sync",
"build:varint": "cd _wasm_varint && deno task --cwd $PWD wasmbuild --sync"
"wasmbuild": "deno run --unstable -A --no-check https://deno.land/x/wasmbuild@0.5.0/main.ts"
```